hackathon/views.py

```diff
@@ -327,3 +432,13 @@ def enroll_toggle(request):
                                 kwargs={'hackathon_id': hackathon_id}))
     else:
         return HttpResponse(status=403)
+
+
+# @login_required
+# def update_hack_awards(request):
+#     if request.method == 'POST':
+#         pass
+#     else:
+#         HackAwardCategoryFormSet = modelformset_factory(
+#             HackAwardCategory, fields=('display_name', 'project'),
+#             formset=HackAwardCategoryFormSet)
```

Comment:
Missing blank line at end of file....
But do we need this last function `def update_hack_awards(request):` if it's all commented-out anyway?

Reply:
Not for this PR, will add another PR to add this view.